projects/Mallard/src/helpers/files.ts

```diff
@@ -139,9 +148,9 @@ export const downloadIssue = (issue: File['id']) => {
             await makeCacheFolder()
             await RNFetchBlob.fs.mv(
                 res.path(),
-                `${issuesDir}/${Date.now()}-${Math.trunc(
-                    Math.random() * 100000,
-                )}.zip`,
+                FSPaths.issueZip(
+                    `${Date.now()}-${Math.trunc(Math.random() * 100000)}`,
```

Comment:
not for this PR, but for this to actually work that'll have to be the issue key

Reply:
Yeah I didn't quite understand what this was doing so I left it ...